v2.1/reference/tools/tidb-lightning/deployment.md

```diff
@@ -170,9 +170,9 @@ You can find deployment instructions in [TiDB Quick Start Guide](https://pingcap
 
 Download the TiDB-Lightning package (choose the same version as that of the TiDB cluster):
 
-- **v2.1.9**: https://download.pingcap.org/tidb-v2.1.9-linux-amd64.tar.gz
-- **v2.0.9**: https://download.pingcap.org/tidb-lightning-v2.0.9-linux-amd64.tar.gz
-- Latest unstable version: https://download.pingcap.org/tidb-lightning-test-xx-latest-linux-amd64.tar.gz
+- **v2.1.9**: `https://download.pingcap.org/tidb-v2.1.9-linux-amd64.tar.gz`
```

Comment:
I don't think we need the inline code format. Without it, users can get it directly by clicking the link. @dcalvin What do you think?

Reply:
Agreed. URLs have their own format already. Inlining them will invalidate the link.